src/components/Stats.tsx

```diff
@@ -0,0 +1,199 @@
+          suppressContentEditableWarning
+        >
+          {this.makeLis(sorted)}
+        </ul>
+      </div>
+    );
+  }
+
+
+  renderChartView(sorted) {
+    const colors = {
+      "production": "#809F69",
+      "testing": "#FFCD61",
+      "cancelled": "#DA5D62"
+    };
+    let chartData = Object.keys(sorted).map((category) => {
+      return { "name": category, "value": sorted[category].length };
+    });
+
+    return (
+        <div className="chart-view">
+          <ResponsiveContainer width={400} height={400}>
+            <PieChart ref="pie-chart-svg">
+              <Legend
+                payload={ chartData.map(
+                  item => ({
+                    id: item.name,
+                    type: "diamond",
+                    value: item.name,
+                    color: colors[item.name],
+                  })
+                )}
+                layout="vertical"
+                height={60}
+                width={200}
+                align="center"
+                verticalAlign="bottom"
+              />
+              <Pie label data={chartData} dataKey="value" nameKey="name" cx="50%" cy="50%" outerRadius={100} />
+              <Tooltip />
+            </PieChart>
+          </ResponsiveContainer>
+          <ResponsiveContainer width={400} height={400}>
+            <BarChart ref="bar-chart-svg" width={400} height={400} data={chartData}>
+              <CartesianGrid strokeDasharray="5 5" />
+              <XAxis dataKey="name" interval={0} />
+              <YAxis />
+              <Tooltip />
+              <Bar dataKey="value" barSize={40} />
+            </BarChart>
+        </ResponsiveContainer>
+      </div>
+    );
+  }
+
+  copy() {
+    let copyArea = (this.statsRef.current as any);
+    copyArea.focus();
+    document.execCommand("selectAll");
+    let copy = document.execCommand("copy");
+    if (copy) {
+      document.execCommand("copy");
+      window.getSelection().removeAllRanges();
+      copyArea.blur();
+      this.setState({ copied: true });
+    }
+  }
+
+  makeLis(sorted) {
+    let total = Object.values(sorted).map(x => (x as any).length).reduce((x, y) => { return x + y; });
+    let getPercentage = (x) => { return (x / total) * 100; };
+    let withStyles = this.state.styled;
+    let categoryInfo = (category) => {
+      let name = category.replace(category[0], category[0].toUpperCase());
+      let length = sorted[category].length;
+      return [
+        <span key={name}>{name}: {length}</span>,
+        <span key={`${name}-${length}`}> ({getPercentage(length)}%)</span>
+      ];
+    };
+
+    return Object.keys(sorted).map((category) => {
+      return (
+        <li key={category} className={withStyles ? "stats-category" : ""}>
+          <section className={withStyles ? "stats-category-name" : ""}>
+            { categoryInfo(category) }
+          </section>
+          <ul className={`${withStyles ? "inner-stats-list " : ""}${this.state.expanded ? "" : "hidden"}`}>
```

Comment:
Maybe you only want to render the list of libraries if the state is expanded, instead of hiding it through css.